wca-guidelines.md

```diff
@@ -207,6 +205,11 @@ To be more informative, each Guideline is classified using one of the following
 - E3d+) [CLARIFICATION] Any stopwatch or watch used by a competitor must not have any functions that would help the competitor find a solution.
 - E3d++) [CLARIFICATION] Competitors should not consider a personal stopwatch or watch as the official time, and must submit their solution when the judge calls "STOP".
 
+## <article-F><clock><clocksolving> Article F: Clock Solving
+
+- F1a1+) [CLARIFICATION] Video or photographic evidence may be used to support this claim.
+- F1a1++) [ADDITION] If the WCA Delegate does not have time to investigate this claim during the competition, they should grant the competitor a provisional extra.
```

Comment:
Could F1a1++ be a generalized regulation with granting provisional extras (ie whenever the Delegate cannot fully investigate regardless of the event / incident).

Reply:
Moved to #1060, therefore removing from this section

```suggestion
```